File: caddisfly-app/app/src/androidTest/java/org/akvo/StripTestSuite.java
package org.akvo;

import org.akvo.caddisfly.instruction.StriptestInstructions;
import org.akvo.caddisfly.navigation.StripTestNavigation;
import org.akvo.caddisfly.test.StriptestTest;
import org.junit.runner.RunWith;
import org.junit.runners.Suite;

// Runs all unit tests.
@RunWith(Suite.class)
@Suite.SuiteClasses({StripTestNavigation.class, StriptestTest.class, StriptestInstructions.class})
public class StripTestSuite {
}

File: caddisfly-app/app/src/androidTest/java/org/akvo/StripTestSuite.kt
package org.akvo

import org.akvo.caddisfly.instruction.StriptestInstructionsTest
import org.akvo.caddisfly.internal.StriptestTest
import org.akvo.caddisfly.test.StriptestSurveyTest
import org.junit.runners.Suite

// Uncomment @Runwith below to run this suite. Commented because gradlew runs the tests twice
// https://github.com/gradle/gradle/issues/2603
//@RunWith(Suite::class)
@Suite.SuiteClasses(StriptestInstructionsTest::class, StriptestTest::class, StriptestSurveyTest::class)
class StripTestSuite
